### PR TITLE
rd

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,136 @@
...
+```
+
+Remember truthy? Here again:
+
+```ruby
+while -1 do # -1 is truthy....
+  puts "say this forever..."
+end
+```
+
```
Comment (maxwellbenton) [inline README.md:67]: Feel like there needs to be a sentence here or a code snippet result 

```
say this forever...
say this forever...
say this forever...
say this forever...
say this forever...
...
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,136 @@
...
+The basic `while...do...end` looks like:
+
+```ruby
+while (condition expression) do
+ # stuff to do
+end
+```
+
+As long as the condition expression is `true` (or truthy), the code inside the
+`do...end` block will run. Keep in  mind, as an _expression_ the condition
```
Comment (maxwellbenton) [inline README.md]: double space between `in  mind`

File: README.md [hunk 3]
```diff
@@ -1,16 +1,136 @@
...
+
+```ruby
+while -1 do # -1 is truthy....
+  puts "say this forever..."
+end
+```
+
+## Use Control-C to Break an Infinitely Looping Program
+
+If you ever accidentally have a program run inside a loop (maybe you made an
```
Comment (maxwellbenton) [inline README.md]: this sentence isn't quite clear. Maybe something like:

If your running a program and find its stuck in an infinite loop

File: README.md [hunk 4]
```diff
@@ -1,16 +1,136 @@
...
+would _always_ be `true` and, therefore, we'd have an infinite loop.
+
+## Terminate a `while...do...end` Loop With `break` Statement
+
+Programmers consider it most "expressive" to put the _condition expression_
+next to the while. Programmers are used to looking next to `while` to find out
+why a loop exited. **However**, we _can_ break out of a loop by using the
+`break` statement.
+
+> **LEARNING NOTE**: You'll also notice we're using a statement modifier and a
```
Comment (maxwellbenton) [inline README.md]: and an

File: README.md [hunk 5]
```diff
@@ -1,16 +1,136 @@
...
+```ruby
+magic_exit_number = 7
+count = 0
+while count < 10 do
+  break if count == magic_exit_number
+  puts "I am the #{count}, I love to count!" # Work
+  count = count + 1
+end
+```
+
```
Comment (maxwellbenton) [inline README.md:143]: I think `magic_exit_number`'s role here this should probably be explained following the code snippet

**Comment by sgharms** [comment]
@maxwellbenton Changes in.

File: README.md [hunk 6]
```diff
@@ -96,13 +110,26 @@ would _always_ be `true` and, therefore, we'd have an infinite loop.
...
+next to the `while`. Programmers are used to looking to the right of the
+`while` to find out why a loop exited. They understand this "why" by
+understanding the _condition expression_. **However**, we _can_ break out of a
+loop by using the `break` statement.
+
+The `while` should cover the general case including when to repeat **and** when
+to finish. A `break` is for priority or anomalous interruption. For example,
+most TV or radio stations have a loop of programming that they run every day.
+But in the case of a special alert bulletin (dramatic weather, emergency, etc.)
+they can "break" out of their regularly loop. The `break` statement is like
```
Comment (maxwellbenton) [inline README.md:122]: regular

**Pull request merged**